src/main/java/xyz/nucleoid/plasmid/mixin/game/event/ScreenHandlerMixin.java

```diff
@@ -64,9 +64,10 @@ private void close(PlayerEntity player, CallbackInfo ci) {
 
     private boolean shouldBlockThrowingItems(PlayerEntity player, int slot, ItemStack stack) {
         ManagedGameSpace gameSpace = ManagedGameSpace.forWorld(player.world);
-        if (gameSpace != null && gameSpace.containsPlayer((ServerPlayerEntity) player)) {
+		ServerPlayerEntity serverPlayer = (ServerPlayerEntity) player;
```

Comment:
```suggestion
	    ServerPlayerEntity serverPlayer = (ServerPlayerEntity) player;
```

Reply:
Fixed in 907e319f91537b25c4a0a522d901d8c4fa82c4df